drivers/adc/adc_stm32.c

```diff
@@ -513,9 +525,10 @@ static int adc_stm32_init(struct device *dev)
 	defined(CONFIG_SOC_SERIES_STM32L0X)
 	LL_ADC_SetClock(adc, LL_ADC_CLOCK_SYNC_PCLK_DIV4);
 #elif defined(CONFIG_SOC_SERIES_STM32F3X) || \
-	defined(CONFIG_SOC_SERIES_STM32L4X)
-	LL_ADC_SetCommonClock(__LL_ADC_COMMON_INSTANCE(),
-			LL_ADC_CLOCK_SYNC_PCLK_DIV4);
+	defined(CONFIG_SOC_SERIES_STM32L4X) || \
+	defined(CONFIG_SOC_SERIES_STM32WBX)
+	LL_ADC_SetCommonClock(__LL_ADC_COMMON_INSTANCE(adc),
+			LL_ADC_CLOCK_SYNC_PCLK_DIV2);
```

Comment:
You're changing argument of the function. This change impacts other series.
If this is a bug you want to fix, please submit as a separate PR and explain your change, so people can have possibility to review  and test.